app/models/event.rb

```diff
@@ -56,7 +56,7 @@ def deadline_has_passed?
   end
 
   def single_player?
-    player_type == Event.player_types[:single]
+    player_type == 'single'
```

Comment:
Is a string 'the way to go' when checking enum equality?

Reply:
http://edgeapi.rubyonrails.org/classes/ActiveRecord/Enum.html

According to docs, the enum attribute returns a string.

Reply:
When I see correctly, there should be an `event.single?` method already

Reply:
correct